tiingo/api.py

```diff
@@ -169,4 +178,7 @@ def get_bulk_news(self, file_id=None):
             url = "tiingo/news/bulk_download"
 
         response = self._request('GET', url)
-        return response.json()
+        if fmt=='json':
+            return response.json()
+        elif fmt=='object':
+            return json.loads(json.dumps(response.json()), object_hook=lambda d: namedtuple('X', d.keys())(*d.values()))
```

Comment:
Same comments as above regarding response.json() and using a full name (maybe `BulkDownload`).